boards/stm32mp157c-dk2/include/periph_conf.h

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2020 Savoir-faire Linux
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     boards_stm32mp157c-dk2
+ * @{
+ *
+ * @file
+ * @brief       Board specific implementations for the STM32MP157C-DK2 board
+ *
+ * @author      Gilles DOFFE <gilles.doffe@savoirfairelinux.com>
+ */
+
+#ifndef PERIPH_CONF_H
+#define PERIPH_CONF_H
+
+/* This board provides an LSE */
+#ifndef CONFIG_BOARD_HAS_LSE
+#define CONFIG_BOARD_HAS_LSE    1
+#endif
+
+/* This board provides an HSE */
+#ifndef CONFIG_BOARD_HAS_HSE
+#define CONFIG_BOARD_HAS_HSE    1
+#endif
+
+#include "periph_cpu.h"
+#include "mp1/cfg_clock_default_208.h"
```

Comment:
You could also adapt [clk_conf.h](https://github.com/RIOT-OS/RIOT/blob/master/cpu/stm32/include/clk/clk_conf.h) and include it here:
```suggestion
#include "clk_conf.h"
```

in `clk_conf.h`, just add the following:

```c
#elif defined(CPU_FAM_STM32MP1)
#include "mp1/cfg_clock_default_208.h"
```
after the l4/l5/wb case and it will be good :)

Reply:
I made it creating `cpu/stm32/include/clk/mp1/cfg_clock_default.h` if its ok for you.